test/test_ops.py

```diff
@@ -1688,12 +1785,40 @@ class TestRefsOpsInfo(TestCase):
 
     @parametrize("op", ref_ops_names)
     def test_refs_are_in_python_ref_db(self, op):
+        err = f"{op} does not have an entry in python_ref_db"
         if op in self.skip_ref_ops:
-            raise unittest.SkipTest(f"{op} does not have an entry in python_ref_db")
-        self.assertIn(op, self.ref_db_names)
+            raise unittest.SkipTest(err)
+        # skip aliases since we don't want them defined in the ref db
+        if op in self.ref_alias_names:
+            return
+        self.assertIn(op, self.ref_db_names, msg=err)
+
+    # like 'test_refs_are_in_python_ref_db' but check in the other direction,
+    # this includes aliases since those are not allowed in 'ref_db_names'
+    @parametrize("op", itertools.chain(ref_db_names, ref_alias_names))
+    def test_python_refs_are_in_ops_names(self, op):
+        # TODO: skip nvfuser refs since those don't have __all__ and it's not
+        # clear at the moment what would be the best way to check those
+        # https://github.com/pytorch/pytorch/issues/85081
+        if op.startswith("ops.nvprims"):
+            return
+        self.assertIn(op, self.ref_ops_names)
+
+    @parametrize("op", ref_db_names)
+    def test_ref_db_has_no_aliases(self, op):
+        self.assertNotIn(op, self.ref_alias_names,
+                         f"Alias {op} in python_ref_db, which is not allowed, "
+                         f"aliases are tested automatically")
 
     @parametrize("op", ref_ops_names)
     def test_refs_are_in_decomp_table(self, op):
+        # skip aliases since those will be registered via their respective ops
```

Comment:
Like with the above I don't think this test needs to be parametrized?

Reply:
This one iterates over each op. It would be messier without the decorator, IMO.